R/g.part1.R

```diff
@@ -274,7 +274,7 @@ g.part1 = function(datadir = c(), outputdir = c(), f0 = 1, f1 = c(),
       # three columns respectively named temperature.offset.x, temperature.offset.y, and temperature.offset.z
       # the end-user can generate this document based on calibration analysis done with the same accelerometer device.
       if (length(params_rawdata[["backup.cal.coef"]]) > 0 & check.backup.cal.coef == TRUE) {
-        bcc.data = read.csv(params_rawdata[["backup.cal.coef"]])
+        bcc.data = data.table::fread(params_rawdata[["backup.cal.coef"]], data.table = FALSE)
         if (isTRUE(params_rawdata[["do.cal"]]) & verbose == TRUE) {
```

Comment:
I think this change is needed, it is used to read the csv with the calibration coefficients from the data_quality_report.csv. If this file has been saved with ";" or "\t" as sep, then it would be good to have a flexible function to read the data rather than read.csv.